charts/steward/values.yaml

```diff
@@ -14,7 +14,6 @@ runController:
     heartbeatInterval: 1m
     heartbeatLogging: true
     heartbeatLogLevel: 3
-    serverRequestTimeout: 1m
```

Comment:
Keep it as documentation with some comment
```
# allowed values are durations. e.g. 1m 
serverRequestTimeout: ""
```

Reply:
done in here https://github.com/SAP/stewardci-core/pull/307/commits/1e0917f87354a27e7ee8e1215eb4b5dacbda451c

Reply:
The documentation is in the README. So I don't think we should double-document parameter.

Reply:
Please remove the line.